install/postflight-check/docker/image.yaml

```diff
@@ -0,0 +1,2 @@
+postflight:
+  postflight: simon3458/primehub-postflight-check:v0.0.4
```

Comment:
nit: could put the image within company wise, or w/o docker in the future.

Reply:
Change it. Done.